isis/scripts/isis3VarInit.py

```diff
@@ -54,4 +91,33 @@
 os.popen("echo 'unset ISISROOT' >>"+isisroot+"/etc/conda/deactivate.d/env_vars.sh")
 os.popen("echo 'unset ISIS3DATA' >>"+isisroot+"/etc/conda/deactivate.d/env_vars.sh")
 os.popen("echo 'unset ISIS3TESTDATA' >>"+isisroot+"/etc/conda/deactivate.d/env_vars.sh")
+>>>>>>> upstream/dev
```

Comment:
Another merge marker

Reply:
It looks like this block was intended to be deleted

Reply:
Yeah.